packages/botfuel-dialog/src/resolver.js

```diff
@@ -31,7 +31,7 @@ class Resolver {
     this.config = config;
     this.kind = kind;
     this.path = `${config.path}/src/${kind}s`;
-    this.localPath = `${__dirname}/${kind}s`;
+    this.kindSdkPath = `${__dirname}/${kind}s`;
```

Comment:
why don't you name it sdkPath (the other path does not have kind in its name)?

Reply:
`sdkPath` whould be `.../botfuel-dialog/src`. If it is something related to the kind, then it should appear somehow in the name IMHO.

Reply:
`path` -> `botPath`
`localPath ` -> `sdkPath`

Reply:
done